server/src/test/scala/controllers/UsersControllerSpec.scala

```diff
@@ -526,4 +528,59 @@ class UsersControllerSpec extends PlayPostgresSpec with LoginUtils {
         error must be("The given email/password doesn't match")
     }
   }
+
+  "POST /users/email-verification-token" should {
```

Comment:
Add more tests: `not send the token if we have an active token created in the last 5 minutes`, `fail if the user is already verified`, `success resending verify email twice`, `success on verifying email using a resend email`

Reply:
I agree with Cristopher, let's add these tests:
- fail if the user is already verified
- success resending verify email twice
- success on logging in after verifying the email that was re-sent